### DISCUSSION TITLE
Delete operation not working in backpack crud

### DISCUSSION THREAD

**Discussion by shilpitrusty** [Laravel-Backpack/community-forum#946] [Q&A (Help)] [answered]
Delete operation from list of user or category or tag etc not working and return 419 error (CSRF token mismatch.). Ho can I fix in backpack admin

**Comment by karandatwani92** [comment, marked as answer]
Hey @shilpitrusty 

Do a `php artisan basset:check`. It will probably tell you where your error is.


My guess would be `APP_URL` in `.env` file.
>Make sure the APP_URL in your .env file is correctly pointing to the URL you use to access your application in the browser, for example: http:127.0.0.1:8000 or http://something.test
